main.css

```diff
@@ -27,22 +26,20 @@ h1 {
   color: black;
   position: relative;
   text-align: left;
-  margin: 20px;
-  left: 35px;
   text-decoration: none;
+  top: 30px;
 }
 h2 {
   text-align: center;
-  padding-top: 100px;
+  padding-top: 50px;
   text-decoration: underline;
```

Comment:
It is not a good from design perspective to underline normal text as it looks as user can think it is a link